core/tee/tee_svc_cryp.c

```diff
@@ -825,6 +858,10 @@ static const struct attr_ops attr_ops[] = {
 		.from_obj = op_attr_bignum_from_obj,
 		.free = op_attr_bignum_free,
 		.clear = op_attr_bignum_clear,
+		.from_user_rsec = op_attr_bignum_from_user_rsec,
```

Comment:
I assume that the `TEE_USAGE_EXTRACTABLE` flag will be used to tell if this function should be called or `op_attr_bignum_from_user()`. How about adding another parameter to `op_attr_bignum_from_user()` instead indicating if the bignum should be initialized as a black key?